tests/python/gaia-ui-tests/gaiatest/tests/functional/cards_view/test_cards_view_kill_apps_with_two_apps.py

```diff
@@ -30,15 +29,12 @@ def test_kill_app_from_cards_view(self):
         self.cards_view.wait_for_cards_view()
 
         # Wait for first app ready
-        self.cards_view.wait_for_card_ready(self._test_apps[1])
+        self.cards_view.cards[1].wait_for_centered()
+        self.assertIn(self.cards_view.cards[1].manifest_url, self.gallery.manifest_url)
 
         # Close the current apps from the cards view
-        self.cards_view.close_app(self._test_apps[1])
-        self.cards_view.close_app(self._test_apps[0])
+        self.cards_view.cards[1].close()
+        self.cards_view.cards[0].close()
 
         # If successfully killed, the apps should no longer appear in the cards view and the "No recent apps" message should be displayed
-        self.assertFalse(self.cards_view.is_app_present(self._test_apps[1]),
-                         "Killed app not expected to appear in cards view")
-
-        self.assertFalse(self.cards_view.is_app_present(self._test_apps[0]),
-                         "Killed app not expected to appear in cards view")
+        self.assertEqual(len(self.cards_view.cards), 0, 'Should have no cards to display')
```

Comment:
I'd change or remove the error message. So if we have more than 0 card, the default error message will show how many we have. We can also change it to reflect how many are displayed:
```py
'Should have no cards to display ({} displayed)'.format(len(self.cards_view.cards))
```